tests/cases/resource_test.py

```diff
@@ -458,20 +458,20 @@ def testGetResourceByPath(self):
         # test is not set
         resp = self.request(path='/resource/lookup',
                             method='GET', user=self.user,
-                            params={'path': '/collection/bogus/path'})
+                            params={'path': '/bogus/path'})
         self.assertStatus(resp, 400)
         # test is set to false, response code should be 400
         resp = self.request(path='/resource/lookup',
                             method='GET', user=self.user,
-                            params={'path': '/collection/bogus/path',
+                            params={'path': '/bogus/path',
```

Comment:
Sorry for the confusion, what I meant in my earlier suggestion was to keep lines 466 and 474 as `/collection/bogus/path`, but only revert the one on line 461 to `/bogus/path` 😄 

Alternatively, you could just add another block altogether to test `/bogus/path` if you wanted to still test `/collection/bogus/path` using the default value for the `test` parameter.